parsel/selector.py

```diff
@@ -62,16 +80,20 @@ def _st(st: Optional[str]) -> str:
 
 
 def create_root_node(
-    text, parser_cls, base_url=None, huge_tree=LXML_SUPPORTS_HUGE_TREE
-):
+    text: str,
+    parser_cls: Type[_ParserType],
+    base_url: Optional[str] = None,
+    huge_tree: bool = LXML_SUPPORTS_HUGE_TREE,
+) -> etree._Element:
     """Create root node for text using given parser class."""
     body = text.strip().replace("\x00", "").encode("utf8") or b"<html/>"
     if huge_tree and LXML_SUPPORTS_HUGE_TREE:
         parser = parser_cls(recover=True, encoding="utf8", huge_tree=True)
-        root = etree.fromstring(body, parser=parser, base_url=base_url)
+        # the stub wrongly thinks base_url can't be None
```

Comment:
Bug report filed

Reply:
Can you share a link for future reference?